src/index.js

```diff
@@ -1,25 +1,34 @@
-import { useState, useEffect } from "react";
+import React, { useState, useRef, useEffect } from "react";
 
 const noop = () => undefined;
 export default function useInputDebounce(
   effect = noop,
-  { initial, delay = 0, minLength = 0 }
+  { initial = "", delay = 0, minLength = 0 }
```

Comment:
if we set a default to "" we cause the effect to be called at the first rendering without typing anything within the input.

Reply:
Removed.